pkg/controllers/disruption/types.go

```diff
@@ -36,6 +36,8 @@ import (
 	"sigs.k8s.io/karpenter/pkg/controllers/provisioning/scheduling"
 	"sigs.k8s.io/karpenter/pkg/controllers/state"
 	"sigs.k8s.io/karpenter/pkg/events"
+	disruptionutils "sigs.k8s.io/karpenter/pkg/utils/disruption"
```

Comment:
Thoughts about just calling this utils since we are already in the disruption package?

Reply:
This isn't in the same overall package, but yes it's mostly to follow convention. I think it's more clear (especially if there are more util packages being used) that we prepend/append it with the type of util it is. Thoughts?